src/pages/main/components/carousel/Carousel.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import styled from 'styled-components';
+import Slider from 'react-slick';
+import { FaCaretLeft, FaCaretRight } from 'react-icons/fa';
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
+
+const settings = {
+  dots: true,
+  infinite: true,
+  speed: 1000,
+  slidesToShow: 1,
+  slidesToScroll: 1,
+  autoplay: true,
+  autoplaySpeed: 5000,
+  prevArrow: <FaCaretLeft />,
+  nextArrow: <FaCaretRight />,
+};
+
+const Carousel = () => {
+  return (
+    <div>
+      <StyledSlider {...settings}>
+        {tumnail.map(item => {
+          return (
+            <div key={item.id}>
+              <ImageContainer>
+                <Img
+                  src={`${item.url}`}
+                  backgroundColor={`${item.background}`}
+                />
```

Comment:
단순 자바스크립트 변수이기 때문에 템플릿 리터럴 사용하지 않으셔도 됩니다.
```suggestion
                <Img
                  src={item.url}
                  backgroundColor={item.background}
                />
```